trackers/huggingface/graphs.py

```diff
@@ -613,3 +626,165 @@ def results_table(df: pd.DataFrame):
         df = df[[model_name in x for x in df["Model Name"]]]
 
     st.dataframe(df, height=min((len(df) + 1) * 35, 35 * 21))
+
+
+def device_funnel(df: pd.DataFrame) -> None:
+    """
+    Show count of how many models compile, assemble, etc
+    """
+    summ = DeviceStageCount(df)
+
+    stages = [
+        "All models",
+        "Convert to ONNX",
+        "Optimize ONNX file",
+        "Converts to FP16",
+        "Builds",
```

Comment:
The way we've defined "build" in the user guide is the process of producing an executable artifact, and then "benchmark" is the process of collecting the perf data. 

It gets a little weird here because Groq involves build+benchmark but the others are just benchmark. Is "Builds + Benchmarks" the correct label for now?

Reply:
Interesting point. The tricky thing about this is that Groq devices only have estimated values, so "benchmark" is theoretically also not correct for Groq devices. Maybe we need a new term here and we can explain it as part of the FAQ or as a footnote?

Reply:
What about `Executes`? Or `Invokes`?

Reply:
Or be more verbose/specific and say `Performance Aquired`?

Reply:
Switching to `Performance Acquired` as suggested